charts/nginx-gateway-fabric/templates/service.yaml

```diff
@@ -17,8 +17,14 @@ spec:
   {{- end }}
 {{- end }}
   type: {{ .Values.service.type }}
-{{- if and (eq .Values.service.type "LoadBalancer") .Values.service.loadBalancerIP }}
+{{- if eq .Values.service.type "LoadBalancer" }}
+  {{- if .Values.service.loadBalancerIP }}
   loadBalancerIP: {{ .Values.service.loadBalancerIP }}
+  {{- end }}
+  {{- if .Values.service.loadBalancerSourceRanges }}
+  loadBalancerSourceRanges:
+{{ toYaml .Values.service.loadBalancerSourceRanges | indent 4 }}
```

Comment:
Looks like this can be indented a bit more. (be sure to re-verify that everything is rendered correctly after the changes)

Reply:
I did verify the changes and it works with `nindent 2` since we have the two extra spaces added in the front of the yaml entries

Reply:
Interesting, a lot of our other fields like this in the helm chart don't account for the leading spaces, but seem to work just fine?

Reply:
Yes interesting, It worked fine with `nindent 4` as well. Maybe the parsing is done accounting for this!